packages/hurl/src/http/client.rs

```diff
@@ -721,6 +724,16 @@ impl Client {
         headers
     }
 
+    /// Parse primary IP into `http::response_ip::IpAddr`
+    fn parse_primary_ip(&mut self) -> Result<IpAddr, HttpError> {
+        match self.handle.primary_ip()? {
+            Some(ip_from_curl) => Ok(IpAddr {
```

Comment:
Also the name of the binding could simply be `Some(ip)` instead of `Some(ip_from_curl)`we can make it concise, and don't need to know that it comes from libcurl